logicrunner/builtin/contract/deposit/deposit.go

```diff
@@ -29,7 +28,8 @@ import (
 type status string
 
 const (
-	confirms uint = 3
+	migrationConfirms       uint                = 3
+	migrationHoldNumOfPulse insolar.PulseNumber = 15768018 //hold on 6 month
```

Comment:
Maybe using `...Offset` name? For example migrationHoldOffset. 
It may be better to make a constant through the constant for a number of months?

Reply:
ok.Thank you